lib/px/block/pxblock.lua

```diff
@@ -163,6 +164,9 @@ function M.load(px_config)
                 local req_query_param = ngx.req.get_uri_args()
                 local enc_url, enc_args
                 local original_req_url = ngx.var.uri
+                if px_config.redirect_to_referer == true then
```

Comment:
Looks like we either way redirect - the why do we need another option of redirect and not just one?
Just raise the question why the original was necessary or it was just something one implemented without thinking about this option, so we can just fix the behavior.